### DISCUSSION TITLE
is there any working example for it? because there is no latest full code example on internet

### DISCUSSION THREAD

**Discussion by ganbaaelmer** [google/model-viewer#3766] [Q&A]
[https://www.reddit.com/user/elmerganbaa/comments/x2uny0/how_to_use_modelviewer_because_there_is_no_full/](url)

https://modelviewer.dev/docs/index.html

in this document there is no full coded example.

if i wrote code it from this reference then its not working correctly in my app.

i mean this website only works and i don't know rest of codes from this website.

```
<!DOCTYPE html>    
<html lang="en">
<head>
 <meta charset="UTF-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <title>model-viewer test</title>
 <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>

 <body style="background-color:rgba(0,0,0,0);"></body>
</head>
<body>
<model-viewer style="width:800px; height: 800px;" id = "glb" crossorigin="anonymous" camera-controls ></model-viewer>
<script>
document.getElementById('glb').src= "https://uzver.metaland.mn/37mb.glb" </script>
</body>
</html>    
```

anyone have full code about model-viewer?

i just need example full code

**How to use model-viewer because there is no full code example on internet?
please help me**

**Comment by elalish** [comment]
You can start with https://glitch.com/edit/#!/model-viewer, from the issue template you deleted 😄
